packages/cli/src/sso/saml/routes/saml.controller.ee.ts

```diff
@@ -149,7 +150,8 @@ export class SamlController {
 				userEmail: loginResult.attributes.email ?? 'unknown',
 				authenticationMethod: 'saml',
 			});
-			throw new AuthError('SAML Authentication failed');
```

Comment:
can we please go back to throwing the auth error, and update tests and frontend accordingly instead?

Reply:
Unless I've missed something, I can't, because both routes that use it need `usesTemplate`, to use `res.render`, which means that the route isn't wrapped with `send`. Unless I stop using `res.render` altogether.